lib/renderer.js

```diff
@@ -20,6 +22,13 @@ $('#save-item-btn').on('click', () => {
   sendItem(item, quantity)
 })
 
+const valueCheck = (title, link) => {
```

Comment:
well after I kept adding empty things from testing, it seemed like a good thing to do, lol :)